styles/style.css

```diff
@@ -70,55 +70,45 @@ svg.sprite-account {
   margin-bottom: 80px;
   line-height: 58px;
```

Comment:
Yoe need to delete line-height or add overflow: hidden. It allow you to avoid scroll for text